### ISSUE TITLE
[Feature] Configurable default `PrizmCardComponent.shadow` value

### ISSUE THREAD

**Issue by AleksandrSibiakov** [zyfra/Prizm#1669]
### Библиотека

- [ ] `@prizm-ui/core`
- [x] `@prizm-ui/components`
- [ ] `@prizm-ui/install`
- [ ] `@prizm-ui/icons`
- [ ] `@prizm-ui/icons-loader`
- [ ] `@prizm-ui/flag-icons`
- [ ] `@prizm-ui/theme`
- [ ] `@prizm-ui/charts`
- [ ] `@prizm-ui/ast`
- [ ] `@prizm-ui/nx-plugin`

### Компонент

_PrizmCardComponent_

### Предложение

_По итогам дизайн ревью требуется убрать тени у всех панелей в приложении_

_Предлагаю сделать ..._

* Поменять дефолтное значение `PrizmCardComponent.shadow` на `none` (по аналогии с `<prizm-card>` внутри `PrizmWidgetComponent`, где используется `none`). 
** Можно совсем убрать инпут `shadow`, а для добавления тени рекомендовать использовать директиву `PrizmShadowDirective` (hostDirectives?)
* Добавить возможность переопределить дефолтное значение тени через токен

**Comment by imonogarov** [comment]
@PrizmDS допустимо ли это с точки зрения дизайна?

**Comment by PrizmDS** [comment]
@imonogarov @ickisIckis по дефолту тень убирать не будем, т.к. для наших проектов она используется всегда на данный момент. 

Добавим переменную для отображения тени

**Comment by ickisIckis** [comment]
@AleksandrSibiakov,  пока нет возможности хорошо прокидывать дефолтное значение от компонента к hostDirectives.
 вот запрос на эту фичу к анугляру: https://github.com/angular/angular/issues/50510, пока даже не запланировано. 

И т.к. мы не можем убрать тень совсем - это не соответствует дизайну и не можем поменять дефолтное значение у 
 самой PrizmShadowDirective (тут неоправданный брейкинг), то пока не целесообразно использовать здесь композицию директив. 

Про токен требуется уточнение - требуется токен именно для Card, верно?

**Comment by AleksandrSibiakov** [comment]
Да. Основная цель для меня - это постараться избежать указания `[shadow]="none"` для каждой `prizm-card` в приложении
